app/jobs/irs_attempt_events_batch_job.rb

```diff
@@ -0,0 +1,31 @@
+class IrsAttemptEventsBatchJob < ApplicationJob
```

Comment:
[job_configurations.rb](https://github.com/18F/identity-idp/blob/main/config/initializers/job_configurations.rb) allows us to set this up to run hourly.

https://github.com/18F/identity-idp/pull/6588/files is an example of another job doing this.